backend/src/main/java/atlantbh/restaurants/services/ReviewService.java

```diff
@@ -25,4 +31,14 @@ public Double getRatingForRestaurant(Long restaurantId) {
             throw new ServiceException(e.getMessage());
         }
     }
+
+    @Override
+    public Review create(Review review) throws ServiceException {
+        Review rev = super.create(review);
```

Comment:
createdReview insted of rev

Reply:
Fixed in 0cd4debab433260ab6a8267aa0b72b31db581edb